oam/install_scripts/post-install

```diff
@@ -206,24 +206,22 @@ fi
 #setup MariaDB Columnstore system logging
 if [ $user = "root" ]; then
 	$installdir/bin/syslogSetup.sh install > /tmp/syslog_install.log 2>&1
-	chmod 777 -R /dev/shm
 	rm -f /etc/default/columnstore
 else
-	$SUDO chmod 777 /tmp
+	$SUDO chmod 755 /tmp
```

Comment:
we shouldn't touch this at all. By default in Linux /tmp is 777 with +t sticky bit for some level of security. This will break things for non-root and other users.